docs/src/index.md

```diff
@@ -162,11 +85,9 @@ the algorithm by specifying a *resource* as defined by
 For example, `imfilter(CPU1(Algorithm.FIR()), img, ...)` would force
 the computation to be single-threaded.
 
-## Arbitrary operations over sliding windows
+### Feature: Models
 
-This package also exports [`mapwindow`](@ref), which allows you to
-pass an arbitrary function to operate on the values within a sliding
-window.
+The `Models` submodule provides predefined image-related models and its solvers that can be reused
+by many image processing tasks.
 
-`mapwindow` has optimized implementations for some functions
-(currently, `extrema`).
+For example, solve the Rudin Osher Fatemi (ROF) model using the primal-dual method.
```

Comment:
Is this a leftover or incomplete? This section seems truncated.

Reply:
untruncated now